crates/settings/src/lib.rs

```diff
@@ -85,6 +85,11 @@ impl Settings {
             ethui_tracing::parse(&self.inner.rust_log)?;
         }
 
+        if let Some(v) = params.get("stacks") {
+            println!("Setting stacks mode to {}", v);
+            self.inner.stacks = serde_json::from_value(v.clone()).unwrap();
```

Comment:
no unwrap please. should end up with a `?` so the error propagates properly

Reply:
Fixed but just as a side note ! The others `from_value` on the same function use `unwrap`